### PR TITLE
Update help page documentation

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the updates on the help page and the contact page.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 0 sec

File: docs/src/user/manual/help-contact.md [hunk 1]
```diff
@@ -23,8 +23,30 @@ The User Guide is a valuable resource for users seeking in-depth guidance.
 1. The User Guide will open in a new browser tab.
 2. A user can browse through the User Guide to find specific information or guidance on various topics related to the platform.
 
-#### Contact Us form
+# Contact Page Documentation
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/help-contact.md:26]:
```suggestion
## Contact Page
```

File: docs/src/user/manual/help-contact.md [hunk 2]
```diff
@@ -23,8 +23,30 @@ The User Guide is a valuable resource for users seeking in-depth guidance.
...
+
+## Description
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/help-contact.md:28]:
```suggestion
### Description
```

File: docs/src/user/manual/help-contact.md [hunk 3]
```diff
@@ -23,8 +23,30 @@ The User Guide is a valuable resource for users seeking in-depth guidance.
...
+
+The Contact Page serves as a means for users to get in touch with your organization or platform. Users can use this page to send messages, inquiries, or feedback. This documentation provides an overview of the fields and features available on the Contact Page.
+
+## Contact Form
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/help-contact.md:32]:
```suggestion
### Contact Form
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Minor heading level changes

**Comment by ketankartoza** [comment]
Thanks @dimasciput

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 17 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 4 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 49 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 10 min

**Pull request merged**